src/main/flight/mixer.c

```diff
@@ -424,44 +425,79 @@ static void updateDynLpfCutoffs(timeUs_t currentTimeUs, float throttle)
+    float motorMixNormalizationFactor;
+    float airmodeTransitionPercent;
+    float motorDeltaScale;
+
+    if (airmodeEnabled || throttle > 0.5f) {
+        airmodeTransitionPercent = 1.0;
+        motorDeltaScale = 0.5f;
+    } else {
+        // this scales the motor mix authority to be 0.5 at 0 throttle, and 1.0 at 0.5 throttle as airmode off intended for things to work.
+        // also lays the groundwork for how an airmode percent would work.
+        airmodeTransitionPercent = scaleRangef(throttle, 0.0f, 0.5f, 0.5f, 1.0f); // 0.5 throttle is full transition, and 0.0 throttle is 50% airmodeTransitionPercent
+        motorDeltaScale = 0.5f * airmodeTransitionPercent; // this should be half of the motor authority allowed
+    }
+
+    motorMixNormalizationFactor = motorMixRange > 1.0 ? airmodeTransitionPercent / motorMixRange : airmodeTransitionPercent;
+
+    const float motorMixDelta = motorDeltaScale * motorMixRange;
+
+    float minMotor = FLT_MAX;
+    float maxMotor = FLT_MIN;
 
     for (int i = 0; i < mixerRuntime.motorCount; ++i) {
-        if (airmodeEnabled || throttle > 0.5f) {
-            if (mixerConfig()->mixer_type == MIXER_LINEAR) {
-                motorMix[i] = scaleRangef(throttle, 0.0f, 1.0f, motorMix[i] + motorMixDelta, motorMix[i] - motorMixDelta);
-            } else {
-                motorMix[i] = scaleRangef(throttle, 0.0f, 1.0f, motorMix[i] + ABS(motorMix[i]), motorMix[i] - ABS(motorMix[i]));
-            }
+        if (mixerConfig()->mixer_type == MIXER_LINEAR) {
+            motorMix[i] = scaleRangef(throttle, 0.0f, 1.0f, motorMix[i] + motorMixDelta, motorMix[i] - motorMixDelta);
+        } else {
+            motorMix[i] = scaleRangef(throttle, 0.0f, 1.0f, motorMix[i] + ABS(motorMix[i]), motorMix[i] - ABS(motorMix[i]));
         }
-        motorMix[i] /= motorMixNormalizationFactor;
+
+        maxMotor = MAX(motorMix[i], maxMotor);
+        minMotor = MIN(motorMix[i], minMotor);
+
+        motorMix[i] *= motorMixNormalizationFactor;
+    }
+
+    // correct throttle so it won't clip any outputs
+    float normalizedMotorMixMax = maxMotor * motorMixNormalizationFactor;
+    float normalizedMotorMixMin = minMotor * motorMixNormalizationFactor;
+    if (normalizedMotorMixMin + throttle < 0.0f) {
+        throttle = -normalizedMotorMixMin;
+    } else if (normalizedMotorMixMax + throttle > 1.0f) {
+        throttle = 1.0f - normalizedMotorMixMax;
     }
 }
 
 static void applyMixerAdjustment(float *motorMix, const float motorMixMin, const float motorMixMax, const bool airmodeEnabled) {
 #ifdef USE_AIRMODE_LPF
     const float unadjustedThrottle = throttle;
     throttle += pidGetAirmodeThrottleOffset();
-    float airmodeThrottleChange = 0;
+    float airmodeThrottleChange = 0.0f;
 #endif
-    
-    const float motorMixNormalizationFactor = motorMixRange > 1.0f ? motorMixRange : 1.0f;
+    float motorMixNormalizationFactor;
+    float airmodeTransitionPercent;
```

Comment:
as above, maybe initialise here as `= 1.0f` then remove line 482 and make 481 `if (!airmodeEnabled && throttle < 0.5f) {`